examples/seismic/acoustic/accuracy.ipynb

```diff
@@ -240,21 +250,55 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 45,
```

Comment:
Starts at 23 and here jump from cell 33 to cell 45. Did you "restart and run all" then save?

Reply:
forgot to restart kernel, just did "run all". thanks